Add a lot more goodies in the mail for all jobs

00ze-cyclone opened the pull request:
# Document the changes in your pull request

Right now most of what you get from the mail is either money (nice but mostly useless) and coins (useless but cool looking).
It is also a pain in the ass to haul all that mail, you can use the disposal system but some outlets are just fucking hidden in corners were nobody ever check so they don't open their mail. We got a mail bag in the code but it just spawn nowhere and can't be made in any ways.
More stuff in the mail might make a few more people want to get the mail, either just for some loot or curiosity. With 3 mail bags in the cargodrobe it'll also finally be possible to a be a mailman without needing a duffebag for the tremendous amounts of mail there is.

# Changelog

General mail goodies:
-merge all the coins into one spawner for coins with a weight of 8, reducing their overall weight in the mail.
-add a donkpocket box spawner, weight of 6
-changed the weight of the 50 credits to 4 from 5
-changed the weight of the 100 credits to 10 from 20

Additions for the job specific mails:

Artist:
-colorful grenade: 10 weight
-spraycan: 10 weight
-instrument beacon: 5 weight
-artistic toolbox: 5 weight
-adaptative paint: 5 weight
-cardboard cutout: 3 weight
-rainbow crayon: 2 weight

Assistant:
-replace the donkpocket box with a donkpocket spawner letting them recieve any flavor of donkpockets instead of just the normal ones

Atmos tech:
-Atmos hardsuit: 1 weight

Captain:
-expensive forgery (fake high value item): 5 weight
-skub: 1 weight
-book of greentext: 1 weight

Chemist:
-metamaterial beaker: 15 weight
-lemoline large beaker: 10 weight
-floor gummy bear: 1 weight

Chief engineer:
-advanced tool: 3 weight
-alien tool: 1 weight

Chief Medical Officer:
-Hippocrate bust: 2 weight

Cook:
-remove the knife
-taster: 2 weight
-whetstone: 1 weight

Curator:
-random book: 20 weight
-random manual: 10 weight
-3 random books: 10 weight
-50 wood planks: 5 weight
-arcane tome: 2 weight
-5 sheets of runed metal: 2 weight
-guide to dank mimery: 1 weight
-book of greentext: 1 weight

Detective:
-remove the .32 trac speedloader and replace it with gutterpunch

Geneticist:
-pill bottle of mutadone: 10 weight
-bottle of unstable mutagen: 5 weight
-Elvis mutator: 1 weight

HoP:
-toner: 10 weight
-four color pen: 10 weight
-APPROVED stamp: 5 weight
-DENIED stamp: 5 weight
-knock-off syndicate rubber stamp: 3 weight
-CC stamp: 1 weight

HoS:
-5 plastic sheets: 20 weight
-hatsky: 10 weight
-fake nuke disk: 5 weight
-liz o' nine tails: 3 weight
-cat o' nine tails: 2 weight
-combat knife: 2 weight
-clusterbang: 1 weight
-oignon grenade: 1 weight

Janitor:
-NT soap: 35 weight
-advanced mop: 5 weight

Lawyer:
-tape: 15 weight
-universal recorder: 10 weight
-megaphone: 10 weight
-gavel: 5 weight
-secHUD (not the sunglasses ones): 5 weight
-banhammer: 1 weight

MD:
-replaced the advanced surgery tools with the spawner with a wieght of 6, decreasing their overall probability
-meme organs: 1 weight

QM:
-gold bar: 10 weight
-toy facehugger: 5 weight
-luxury elite bar capsule: 2 weight
-RCD: 2 weight
-donksoft board: 1 weight

RD:
-deleted the monkey cubes
-neutral AI law board: 15 weight
-tank transfer valve: 15 weight
-safe AI law board: 10 weight
-toy facehugger: 5 weight
-alien surgery tool: 2 weight
-alien tool: 2 weight
-random organ: 2 weight
-warpcrystal: 1 weight

Roboticist:
-removed the 20 metal sheets
-50 sheets of metal: 15 weight
-IPC revival board: 10 weight
-20 plasteel sheets: 5 weight
-bluespace cell: 5 weight
-refined dilithium crystal: 5 weight

Scientist:
-bluespace crystal: 10 weight
-megaphone: 10 weight
-anomaly neutralizer: 5 weight
-strange object: 5 weight

Secoff:
-teargas grenade: 4 weight
-flashbang: 2 weight
-SWAT mask: 1 weight

Engineer:
-removed the light replacer and box of lights
-50 sheets of metal: 25 weight
-50 sheets of glass: 25 weight
-20 sheets of plasteel: 5 weight
-made the cool red hard hat have a weight of 2 instead of 1
-engineering hardsuit: 1 weight

Virologist:
-fake GBS bottle: 4 weight
-magnitis bottle: 3 weight
-pierro throat bottle: 3 weight
-jitter bottle: 3 weight
-anxiety bottle: 3 weight

Warden:
-removed the cigarette pack

Brig phys:
-first aid kit: 20 weight
-atropine autoinjector: 10 weight
-hypospray brute kit: 10 weight
-hypospray burn kit: 10 weight
-medicated suture: 5 weight
-advanced mesh: 5 weight
-pepperspray: 4 weight

Clerk:
-3 random maint loot: 35 weight
-5 sheets of plastic: 30 weight
-random plushie: 20 weight
-minimeteor toy: 15 weight
-slot machine board: 15 weight
-toy desword: 10 weight
-toy His Grace: 10 weight
-syndicate smokes: 5 weight
-hugbox: 5 weight
-random lipstick: 5 weight
-skub: 2 weight
-warpcrystal: 1 weight

Mining medic:
-advanced hypospray kit: 5 weight

Network admin:
-pizza box: 10 weight
-experimental neural processing unit: 5 weight

Psychiatrist:
-hugbox: 30 weight
-mannitol pill bottle: 25 weight
-happiness pill bottle: 10 weight
-stimulant pill bottle (not actual stimulants): 10 weight
-morphine bottle: 5 weight
-flash: 4 weight

Tourist:
-random plushie: 15 weight
-expensive forgery (fake high value item): 5 weight
-binoculars: 5 weight
-photo album: 4 weight
-sunglasses: 1 weight

:cl:  
rscadd: Added a lot of possible things in the mail
rscadd: Added the mail bags in the cargodrobe
tweak: reduced the chances of getting coins or money in the mail
/:cl:

00ze-cyclone committed:
MAIL

00ze-cyclone committed:
Merge branch 'yogstation13:master' into moar-mail

00ze-cyclone committed:
the holy bag

00ze-cyclone committed:
Merge branch 'yogstation13:master' into moar-mail

00ze-cyclone committed:
fix?

MajManatee reviewed (approved):
im iffy on some of these items, but its fine mostly.

Moltijoe reviewed (commented):
Couple of things

ReddicusDragon reviewed (dismissed):
pain.

00ze-cyclone committed:
Update code/modules/jobs/job_types/research_director.dm

Co-authored-by: Redd <36747151+ReddicusDragon@users.noreply.github.com>

00ze-cyclone committed:
changes

Thank you Redd for fucking loosing your mind over the most inane shit like stamps or a funny noise hammer or just repeat what Molti said but with additional irrespect. Really make me want to loose hours of my life for this shit.

00ze-cyclone committed:
fix

00ze-cyclone committed:
Merge branch 'master' into moar-mail

00ze-cyclone committed:
Update dynamic.json

00ze-cyclone committed:
Merge branch 'moar-mail' of https://github.com/00ze-cyclone/Yogstation into moar-mail

00ze-cyclone committed:
guh

00ze-cyclone committed:
Update artist.dm

guh

00ze-cyclone committed:
Update artist.dm

finally

Moltijoe reviewed (approved):
code checks out, balance is still very questionable

00ze-cyclone committed:
Merge branch 'master' into moar-mail

00ze-cyclone committed:
I had more ideas
